Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -5055,6 +5055,59 @@ public void Actions_Vector2Composite_RespectsButtonPressurePoint()
         Assert.That(value.Value, Is.EqualTo(Vector2.left));
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_Vector2Composite_ReportsCorrectControlInCallback()
```

Comment:
Where possible, please avoid "correct" and such in test names. Maybe `Actions_CompositesReportControlThatTriggeredTheCompositeInCallback` or something like that.